packages/medusa/src/types/store.ts

```diff
@@ -13,6 +13,23 @@ export type UpdateStoreInput = {
   default_sales_channel_id?: string
 }
 
+/**
+ * @schema ModulesResponse
```

Comment:
Question: why is that in store types? what does it correspond to?

Reply:
That was in `./modules.ts`, it was moved to the new package but the OAS is not recognizing it there, for now I'm just declaring it here but we will have to revisit OAS generation working with modules/packages.

Reply:
Ping me when you want to change the OAS generator. I'll be able to help.